index.html

```diff
@@ -10,43 +10,66 @@
 <link rel="stylesheet" href="styles.css">
 <body>
     <header class="header">
-        <a href="#" class="nabar-brand"> My logo</a>
-    <nav class="navbar">        
-        <i class="fa fa-bars" class="menu-icon"></i>        
+        <a href="#" class="navbar-brand"> My logo</a>
+    <nav class="navbar">
+        <div class="hamburger">        
+            <i class="fa fa-bars" class="menu-icon"></i>
+        </div>
+        
```

Comment:
good use of html semantictags, you can also use better class names for easy reference

Reply:
- [ ] Excellent work using GitHub features to perform this review. There are a few more things to work in order to give good-quality reviews. 
 - You should phrase your comments politely as suggestions instead of orders for example "**I think it would be better to use better class names for easy reference"**. The use of terms like **please** and **kindly** also help to make you sound polite 😄 
 - You can also give 1 or 2 examples of what you consider better class names so that the student gets the idea. 👍🏾